Automattic/crowdsignal-ui
# New/phone number block

Created a new block for Phone Number, complete with a drop down country icon selector.

To test:

- Apply D88654 backend changes
- Add a phone number block
- Test the settings (default country code, whether the countries appear or not)
